test/unit/BCoWPool.t.sol

```diff
@@ -130,3 +148,84 @@ contract BCoWPool_Unit_EnableTrading is BaseCoWPoolTest {
     bCoWPool.enableTrading(appData);
   }
 }
+
+contract BCoWPool_Unit_Verify is BaseCoWPoolTest {
+  function test_Revert_NonBoundToken() public {
```

Comment:
What if you fuzz the token addresses? This is the way we did it in the bPool tests. You will need to assume not to be a valid token

Reply:
Is it okay to fuzz the non-bound token address but keep configuring the bCoWPool with the same 3 hard-coded tokens from `Utils.sol`? and in a more general sense, is it okay to fuzz the _inputs_ of a test but not fuzz its _setup_ ?

Reply:
Yep, ideally we would fuzz everything. We had to hardcode the 2 tokens because it was pushing foundry to it's limits (because too many variables to fuzz). But I like how you did it now.

Reply:
yeah, and we don't expect much variations in the behaviour on the ultimate token address being used, adding 2 more vars to fuzz to the setup seems like an overkill total